lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -7,6 +7,7 @@
 module ApplianceConsole
   CERT_DIR = ENV['KEY_ROOT'] || RAILS_ROOT.join("certs")
   KEY_FILE = "#{CERT_DIR}/v2_key"
+  KEY_FILE_BACKUP = "#{CERT_DIR}/vk.bak"
```

Comment:
I think backup file should be named something like `v2_key.backup`.
Also, I would suggest to freeze string constant (as Travis recommends)